include/internal/catch_approx.hpp

```diff
@@ -44,7 +44,7 @@ namespace Detail {
             approx.scale( m_scale );
             return approx;
         }
-#if defined(__cplusplus) && __cplusplus >= 201103L
+#if CATCH_CPP11_OR_GREATER
```

Comment:
Why not "[Why you should prefer #ifs over #ifdefs](https://www.iar.com/support/resources/articles/advanced-preprocessor-tips-and-tricks/)" (somewhat past halfway)  :gem: